drafts/draft-0005.md

```diff
@@ -0,0 +1,188 @@
+## Preamble
+
+```
+SEP: <to be assigned>
+Title: Bootstrapping Multisig Coordination
+Author: Paul Selden <paul.selden@stellarguard.me>, Nikhil Saraf <nikhil@interstellar.com>
+Status: Draft
+Created: 2018-10-30
+```
+
+## Simple Summary
+
+Provides a standard way for multisig accounts to designate where transactions should be submitted for coordination of additional signatures.
+
+## Abstract
+
+An account may designate which multisig server to use in a data entry that points to a server with a `stellar.toml` file. This file contains a `MULTISIG_ENDPOINT` field which is an API where multisig transactions should be submitted. Wallets should implement this SEP so they can support multisig accounts without requiring them to run a multisig service themselves.
+
```

Comment:
There's a big hole: how am I suppose to actually get pending transactions? Coordination have two directions: how do I know my co-signers did send transactions to the coordinator?

Reply:
Would be nice to be able to review the whole design, rather than just half of it.

Reply:
I think you didn't got it: how am I getting to display pending transaction **in my wallet**? If my wallet can't get pending transactions from a multisig service how do I offer user to sign them?

Reply:
I’ll say it again.. This SEP **does NOT** describe _how a user will sign **pending transactions** or **signature requests** or **multisig requests**_ or anything else of that nature. That falls into the area of **how to do multisig coordination**, which this SEP is **strictly not**.

What this SEP does describe is **how to send a signature request to an external multisig service**.

Reply:
Well then, why is an endpoint to get tx status specified in that SEP?

I'm sorry you have to use a lot of those dark characters here but maybe the lack of clarity is not only from my side, you know.

Reply:
The [HTTP specification for a POST request](https://www.w3.org/Protocols/rfc2616/rfc2616-sec9.html#sec9.5) species the following:
> If a resource has been created on the origin server, the response SHOULD be 201 (Created) and contain an entity which describes the status of the request and refers to the new resource, and a Location header (see section 14.30).

To comply with the HTTP POST request semantics we need to provide a link to fetch the _status_ of this newly created resource and also therefore need to support a **status** endpoint to allow _retrieval of this newly created resource_.

Reply:
Yeah agreed this part _is_ coordination specific, but only in the "let's do the minimum work to make things functional and then we can explore additions from there based on real use cases".

Reply:
we can revisit this later if needed.